Library/Formula/jemalloc.rb

```diff
@@ -4,6 +4,7 @@ class Jemalloc < Formula
   url "http://www.canonware.com/download/jemalloc/jemalloc-3.6.0.tar.bz2"
   sha256 "e16c2159dd3c81ca2dc3b5c9ef0d43e1f2f45b04548f42db12e7c12d7bdf84fe"
   head "https://github.com/jemalloc/jemalloc.git"
+  revision 2
```

Comment:
You've bumped the revision too hard here. The default revision is `0` and is unwritten, so when bumping it gets increased to `1` rather than `2`.